tools/build/setnextversion.sh

```diff
@@ -16,60 +16,125 @@
 # specific language governing permissions and limitations
 # under the License.
 
-version='TESTBUILD'
-sourcedir=~/cloudstack/
-branch='master'
+set -e
 
-usage(){
-    echo "usage: $0 -v version [-b branch] [-s source dir] [-h]"
-    echo "  -v sets the version"
-    echo "  -b sets the branch (defaults to 'master')"
-    echo "  -s sets the source directory (defaults to $sourcedir)"
-    echo "  -h"
+usage() {
+    cat << USAGE
+Usage: setnextversion.sh --version string [OPTIONS]...
+Set the next version of CloudStack in the POMs.
+
+Mandatory arguments:
+   -v, --version string                    Set the next version to be applied
+
+Optional arguments:
+   -b, --branch string                     Set the branch to update the version into (default "master")
+   -s, --sourcedir string                  Set the source directory to clone repo into (default "$sourcedir")
+   -n, --no-commit                         Apply only the version change and don't git commit them (default "false")
```

Comment:
@khos2ow @rafaelweingartner please update the wiki as well.

Reply:
Where in wiki? The only place I could find that references `setnextversion.sh` is https://cwiki.apache.org/confluence/display/CLOUDSTACK/Release+Procedure .

Reply:
As far as I know we do not have a wiki page to document this script. It might be interesting to create one though.